src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -70,6 +73,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         if (null === $name = $input->getArgument('name')) {
             $this->listBundles($errorIo);
+
+            $kernel = $this->getApplication()->getKernel();
+            if ($kernel->getAlias()
```

Comment:
shouldn't this call be after the instanceof check? (same in the other command)

Reply:
Yes, done.